arangod/VocBase/Methods/Collections.cpp

```diff
@@ -287,6 +290,11 @@ Result Collections::create(TRI_vocbase_t& vocbase,
     helper.add(arangodb::StaticStrings::DataSourceName,
                arangodb::velocypack::Value(info.name));
 
+    if (useRevs) {
+      helper.add(arangodb::StaticStrings::UsesRevisionsAsDocumentIds,
```

Comment:
@graetzer So from what I can tell, this should be the right way to add a flag for an actual collection creation, not counting when we spin up a new shard for replication. Does this seem robust enough?

Reply:
in the cluster case, yes that should be sufficient

Reply:
@graetzer Is there a different code path to be concerned about in the single server case?